### PR TITLE
lido liquidity test incremental model

### PR DESCRIPTION
**Pull request by ppclunghe**
Test model, don't merge please

### CODE REVIEW

**Comment by ppclunghe** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_balancer_pools.sql [hunk 1]
```diff
@@ -85,8 +75,15 @@ WHERE call_create.output_0 in (select distinct  poolAddress from pools)
...
         decimals,
         avg(price) AS price
-    FROM {{ source('prices', 'usd') }}
-    WHERE date_trunc('day', minute) >= date '{{ project_start_date }}'
+    FROM {{ source('prices', 'usd') }} p
+    
+    {% if not is_incremental() %}
+    WHERE DATE_TRUNC('day', p.minute) >= DATE '{{ project_start_date }}'
+    {% endif %}
+    {% if is_incremental() %}
```
Comment (jeff-dude) [inline models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_balancer_pools.sql:83]: minor: can replace the end/if with just `else`, so it's one line vs. two

Comment (jeff-dude) [reply models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_balancer_pools.sql:83]: will only comment here for that

File: models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_balancer_pools.sql [hunk 2]
```diff
@@ -324,25 +358,31 @@ on main.day = paired4.day and main.pool_id = paired4.pool_id
         poolId,
         sum(case when tokenOut = 0x5979d7b546e38e414f7e9822514be443a4800529 then p.price*amountOut/1e18 
              else p.price*amountIn/1e18 end) as trading_volume
-    from {{source('balancer_v2_arbitrum','Vault_evt_Swap')}} s
+    from delta_prod.balancer_v2_arbitrum.Vault_evt_Swap s
```
Comment (jeff-dude) [inline models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_balancer_pools.sql:361]: please switch back to jinja syntax

**Review by jeff-dude — CHANGES_REQUESTED** [review]
i focused only on `lido_liquidity_arbitrum_balancer_pools.sql` for initial review, assuming all others (other than final cross-chain spell) follow a similar pattern of changes. i left a few minor notes in review.

overall, the idea is:
- each upstream spell is an incremental model, which would then be refreshed hourly
- the final cross-chain spell is a view, which starts with the dates CTE to get a running list of dates since project start, then left joins the upstream spells to get up-to-date data on the incremental refresh schedule

have you been able to thoroughly test the output CI tables to ensure the data is as you expected in the new setup?

**Comment by ppclunghe** [comment]
@jeff-dude thanks a lot, your description of the incremental model is correct 
I checked the output CI tables, data seems to be correct, fixed endif/else issue in all spells

**Review by jeff-dude — APPROVED** [review]
let's give this a go!

thanks for taking this initiative on

**Pull request merged**